lookout/style/typos/tests/test_symspell.py

```diff
@@ -18,8 +21,12 @@ def test_compare(self):
 class SymSpellTest(unittest.TestCase):
     def test_generate_candidates(self):
         symspell = SymSpell(max_dictionary_edit_distance=1)
-        symspell.load_dictionary(VOCABULARY_FILE)
-        self.assertEqual(symspell.lookup("ofset", 0, 1)[0].term, "offset")
+        with tempfile.NamedTemporaryFile() as vocabulary_file:
+            with smart_open(VOCABULARY_FILE, "r") as compressed:
+                with open(vocabulary_file.name, "w") as f:
+                    f.write(compressed.read())
+            symspell.load_dictionary(vocabulary_file.name)
```

Comment:
Can you please change the implementation of `symspell.load_dictionary` to avoid the temporary file

```python
def load_dictionary(self, corpus):
    for line in corpus:
        key, count = line.split()
        count = int(count)
        self._create_dictionary_entry(key, count)
    if self._deletes...
```

Reply:
Update: use `csv.reader`

Reply:
It's a bug, I put smart_open there (in symspell) and removed tempfile everywhere except this place.